src/hypothesis/internal/conjecture/engine.py

```diff
@@ -147,8 +134,17 @@ def test_function(self, data):
             data.freeze()
             self.note_details(data)
 
+        self.target_selector.add(data)
+
         self.debug_data(data)
-        if data.status >= Status.VALID:
+
+        tags = frozenset(
+            self.tag_intern_table.setdefault(t, t)
+            for t in data.tags
+        )
+        data.tags = self.tag_intern_table.setdefault(tags, tags)
+
+        if data.status == Status.VALID:
```

Comment:
I don't follow why this changed from `>=` to `==` - is this just a matter of style and future proofing, or has the Status enum actually changed?

Reply:
I confess I don't remember for sure. It's not a question of the enum having changed though - the idea would be to exclude interesting examples from this bit.

Attempting to backfill the reasoning, I think the idea is that we don't actually want to count interesting examples for our coverage tracking, because interesting examples may hit weird and surprising corners of coverage by virtue of their error, so they don't work well as a representative example.